docs/basics/complete_example.md

```diff
@@ -47,7 +46,7 @@ doesn't either.
 
 Note how here, we're not returning a dataframe to the user - we just take a dataframe in, and
 store some internal state. Therefore, we use `nw.from_native` explicitly, as opposed to using the
-utility `@nw.narwhalify_method` decorator.
+utility `@nw.narwhalify` decorator.
```

Comment:
we can now use `narwhalify` for this example, right? if it returns `None`, and the default is `strict=False`, then I think it'll still work? it might be easier to teach to use narwhalify everywhere?